src/SFA.DAS.AdminService.Web/Infrastructure/RoatpApplyTokenService.cs

```diff
@@ -1,28 +1,29 @@
 ﻿using Microsoft.Azure.Services.AppAuthentication;
 using Microsoft.Extensions.Hosting;
 using SFA.DAS.AdminService.Settings;
+using System.Threading.Tasks;
 
 namespace SFA.DAS.AdminService.Web.Infrastructure
 {
     public class RoatpApplyTokenService : IRoatpApplyTokenService
     {
         private readonly IWebConfiguration _configuration;
-        private readonly IHostingEnvironment _hostingEnvironment;
-        public RoatpApplyTokenService(IWebConfiguration configuration, IHostingEnvironment hostingEnvironment)
+        private readonly IHostEnvironment _hostEnvironment;
+        public RoatpApplyTokenService(IWebConfiguration configuration, IHostEnvironment hostingEnvironment)
```

Comment:
I made this change because `IHostingEnvironment` is now deprecated, and the suggestion is to use `IHostEnvironment`.